pitch.md

```diff
@@ -23,38 +89,146 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
 by describing which unicorn properties (horn color, tattoo, etc.) are most 
 important to users and what kinds of values those properties might have._ 
 
+The user should be able to describe elements of a specific graph that they want.
+They will be able to generate very commonly used elements of economics graphs, such
+as supply and demand curves, equilibrium point, production possibilities curves,
+and so on, with a single function call. Then, they would be able to customize
+each of these built-in elements if needed, with things such as slope and
+intercepts being passed in as optional parameters. 
+
+In addition, there would be other properties they can generate such as axes
+values, titles, axes intersection points, and so on.  I could imagine further, 
+more specific customizations being possible to give the DSL an almost secondary
+purpose as a graphing language; however, we'd want to focus and make it as easy
+as possible for users to create graphs for economics specifically first.
+
 
 ## Operation (semantics)
 _What might happen when a program runs? Is there a computational model that
 corresponds with your domain? How does a program interact with the
 user? What kinds of semantic (i.e., non-syntax) errors might occur, and how 
 might they be communicated to the user?_
+A program would run when a document is saved. When this happens, we'd expect the 
+DSL code to be replaced with an image of the graph that was specified. 
```

Comment:
Another possibility would be to show a preview next to the DSL code, and continue displaying the DSL code as well. I'm not sure which is better. Your way might make the notes easier to read and enable better feedback to the note taker. My way has the advantage that it is easy to go back and edit the graphing code if you make a mistake. This might be a trade off, unless you can figure out a way to get the best of both.